client/src/pages/Profile/ProfileListings/ProfileListings.tsx

```diff
@@ -0,0 +1,137 @@
+            />
+            <Grid item xs={12}>
+                <Grid container direction="column">
+                    <Grid item>
+                        <Box justifyContent="center" >
+                            <Typography variant="h4" align="center">Your search results</Typography>
+                            <Box display="flex" justifyContent="center" mt={3}>
+                                <TextField 
+                                    label="Toronto, Ontario" 
+                                    margin="normal" 
+                                    variant="outlined" 
+                                    InputProps={{
+                                        startAdornment: (
+                                            <InputAdornment position="start">
+                                            <Search />
+                                            </InputAdornment>
+                                        ),
+                                    }}
+                                />
+                                <TextField 
+                                    margin="normal" 
+                                    variant="outlined" 
+                                    type="date"
+                                />
+                            </Box>
+
+                            <Box display="flex" flexWrap="wrap" justifyContent="center" pt={5} className={classes.profileArea}>
+                                {
+                                    data.map((d, index) => {
+                                        return(
+                                            <Link key={index} to="/profile-details" style={{textDecoration: 'none'}}>
+                                                <Card className={classes.cardWrapper}>
+                                                    <CardActionArea>
+                                                        <Avatar className={classes.cardAvatar} alt="Remy Sharp" src={d.image_url} />
+                                                        <CardContent>
+                                                        <Typography gutterBottom variant="h5" align="center" component="h2">
+                                                            {d.name}
+                                                        </Typography>
+                                                        <Typography gutterBottom variant="body1" align="center" component="h2">
+                                                            {d.catch_phrase}
+                                                        </Typography>
+                                                        <Typography align="center">
+                                                            <Rating readOnly/>
+                                                        </Typography>
+                                                        <Typography align="center" variant="body2" color="textSecondary" component="p">
```

Comment:
Just curious about this: 
Is there a particular reason we specify the component in these `Typography` components? I would have thought using just the variant would be enough in most cases.